src/afl-showmap.c

```diff
@@ -1011,14 +884,15 @@ int main(int argc, char **argv_orig, char **envp) {
 
   afl_shm_deinit(&shm);
 
-  u32 ret = child_crashed * 2 + fsrv->child_timed_out;
+  u32 ret = child_crashed * 2 + fsrv->last_run_timed_out;
 
   if (fsrv->target_path) ck_free(fsrv->target_path);
 
   afl_fsrv_deinit(fsrv);
   if (stdin_file) ck_free(stdin_file);
 
   argv_cpy_free(argv);
+  if (fsrv->qemu_mode) free(use_argv[2]);
```

Comment:
How about `wine_mode`? Probably has the same leak

Reply:
if wine_mode is set also qemu_mode is set

Reply:
in the code if wine mode is used it also sets qemu_mode. hence this is all that is needed